helm/observability-operator/templates/alertmanager/_helpers.tpl

```diff
@@ -0,0 +1,5 @@
+{{/* vim: set filetype=mustache: */}}
+
+{{- define "alertmanager-secret.name" -}}
+{{- include "resource.default.name" . -}}-alertmanager
```

Comment:
how about?

```suggestion
{{- include "resource.default.name" . -}}-alertmanager-template
```

Reply:
The secret will hold both the config and the templates, so the name should stay as `-alertmanager`